commit 69cc703041948b408b791729522f813a4f27b776 (refs/changes/75/2475/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-08-29T20:53:10+02:00 (2 years, 1 month ago)
    
    MANTA-3377 COAL config should default to 3 storage zones
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/lib/adm.js b/lib/adm.js
index f1cee61..9945601 100644
--- a/lib/adm.js
+++ b/lib/adm.js
@@ -2130,7 +2130,7 @@ maAdm.prototype.dumpConfigCoal = function (options, callback)
 	    'postgres': 2,
 	    'moray': 1,
 	    'electric-moray': 1,
-	    'storage': 2,
+	    'storage': 3,
 	    'authcache': 1,
 	    'webapi': 1,
 	    'loadbalancer': 1,
